web/components/login/SignedIn.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { firebase } from '../../firebase';
+
+export default class SignedIn extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: null,
+    };
+  }
+
+  componentWillMount() {
+    this.removeAuthListener = firebase.auth().onAuthStateChanged((user) => {
```

Comment:
did we decide on promises vs async-await?

Reply:
nope